### ISSUE TITLE
Pricing Method field is empty on the reference picker

### ISSUE THREAD

**Issue by githubbob42** [githubbob42/mingle2github2#3025]
**[Mingle Card: 3324](https://liquidframeworks.mingle.thoughtworks.com/projects/alpine_mobile/cards/3324)**
```
| | |
|-|-|Dependent on both

https://github.com/Liquidframeworks/alpine-mobile/pull/1441

https://github.com/Liquidframeworks/alpine-fetch/pull/70||[Push to Site fxqa1](http://buildslave3.basecamp.liquidframeworks.com:5000/requestQAbuild?qaSite=fxqa1&repoUrl=https%3A%2F%2Fgithub.com%2Fmramad2%2Falpine-mobile&branch=pricing-method-fixes&repoName=alpine-mobile&repoOwner=Liquidframeworks&cards=%5B%7B%22projectName%22%3A%22alpine_mobile%22%2C%22cardNumber%22%3A%223324%22%7D%5D&requestNumber=1442&requestUrl=https%3A%2F%2Fgithub.com%2FLiquidframeworks%2Falpine-mobile%2Fpull%2F1442)||[Push to Site fxqa2](http://buildslave3.basecamp.liquidframeworks.com:5000/requestQAbuild?qaSite=fxqa2&repoUrl=https%3A%2F%2Fgithub.com%2Fmramad2%2Falpine-mobile&branch=pricing-method-fixes&repoName=alpine-mobile&repoOwner=Liquidframeworks&cards=%5B%7B%22projectName%22%3A%22alpine_mobile%22%2C%22cardNumber%22%3A%223324%22%7D%5D&requestNumber=1442&requestUrl=https%3A%2F%2Fgithub.com%2FLiquidframeworks%2Falpine-mobile%2Fpull%2F1442)||[Push to Site fxqa3](http://buildslave3.basecamp.liquidframeworks.com:5000/requestQAbuild?qaSite=fxqa3&repoUrl=https%3A%2F%2Fgithub.com%2Fmramad2%2Falpine-mobile&branch=pricing-method-fixes&repoName=alpine-mobile&repoOwner=Liquidframeworks&cards=%5B%7B%22projectName%22%3A%22alpine_mobile%22%2C%22cardNumber%22%3A%223324%22%7D%5D&requestNumber=1442&requestUrl=https%3A%2F%2Fgithub.com%2FLiquidframeworks%2Falpine-mobile%2Fpull%2F1442)||[Push to Site fxqa4](http://buildslave3.basecamp.liquidframeworks.com:5000/requestQAbuild?qaSite=fxqa4&repoUrl=https%3A%2F%2Fgithub.com%2Fmramad2%2Falpine-mobile&branch=pricing-method-fixes&repoName=alpine-mobile&repoOwner=Liquidframeworks&cards=%5B%7B%22projectName%22%3A%22alpine_mobile%22%2C%22cardNumber%22%3A%223324%22%7D%5D&requestNumber=1442&requestUrl=https%3A%2F%2Fgithub.com%2FLiquidframeworks%2Falpine-mobile%2Fpull%2F1442)||[Push to Site fxsignoff1](http://buildslave3.basecamp.liquidframeworks.com:5000/requestQAbuild?qaSite=fxsignoff1&repoUrl=https%3A%2F%2Fgithub.com%2Fmramad2%2Falpine-mobile&branch=pricing-method-fixes&repoName=alpine-mobile&repoOwner=Liquidframeworks&cards=%5B%7B%22projectName%22%3A%22alpine_mobile%22%2C%22cardNumber%22%3A%223324%22%7D%5D&requestNumber=1442&requestUrl=https%3A%2F%2Fgithub.com%2FLiquidframeworks%2Falpine-mobile%2Fpull%2F1442)||[Push to Site fxsignoff2](http://buildslave3.basecamp.liquidframeworks.com:5000/requestQAbuild?qaSite=fxsignoff2&repoUrl=https%3A%2F%2Fgithub.com%2Fmramad2%2Falpine-mobile&branch=pricing-method-fixes&repoName=alpine-mobile&repoOwner=Liquidframeworks&cards=%5B%7B%22projectName%22%3A%22alpine_mobile%22%2C%22cardNumber%22%3A%223324%22%7D%5D&requestNumber=1442&requestUrl=https%3A%2F%2Fgithub.com%2FLiquidframeworks%2Falpine-mobile%2Fpull%2F1442)| |
```

Dev Notes
---------

It apears that this isn't specific to pricing method but one variation of the problem. No other picklist value is displayed on any compact layout. pricebook picker, account picker. etc....

However, because of the way we alias pricing method on the server, even if the other issue with picklist values is resolved, an additional fix is needed to get pricing method to show.  

The picklist fixes have been split into another card(#3360) and the pricing method fix left with this card. card #3660 must be merged and rebased before we can attempt to test this fix.  

Steps to reproduce
------------------

```
| | |
|-|-|
|**Version #**|
|Prod|
|**OS**|
| |
|**Browser**|
| |
|**Username**|
|ajf@t3.com|
|**Password**|
|computer47|

  

 


```

1.  Create a ticket and click to add ticket item
2.  Select any ticket item
3.  Scroll all the way over to the right and look at the picklist field 'Pricing Method' in the price book item picker window, not the ticket item window.

Expected result
---------------

Should display value like 'Straight Pricing'

Actual Result
-------------

Its empty

Related Cards
-------------

card #3360

Test Plan
---------

1.  Add Pricing Method to Price book item search layout
2.  Sync 
3.  Navigate to Add to ticket screen
4.  Select Pricebook tab it not already selected and validate that the pricing method value now appears for each Price book item
5.  Add item
6.  Sync
7.  Validate that the recently added ticket item shows up, and the PBI Pricing Method field on the ticket item has inherited the price book item's pricing method
8.  Validate that other PBI fields on the ticket item have inherited the values from it's price book item.

**Comment by githubbob42** [comment]
**Pull Request #1442**
